weed/shell/command_ec_encode.go

```diff
@@ -119,7 +119,7 @@ func (c *commandEcEncode) Do(args []string, commandEnv *CommandEnv, writer io.Wr
 
 	// encode all requested volumes...
 	for _, vid := range volumeIds {
-		if err = doEcEncode(commandEnv, *collection, vid); err != nil {
+		if err = doEcEncode(commandEnv, *collection, vid, *parallelize); err != nil {
```

Comment:
how about using an integer of the number of paralleling goroutines?

Reply:
Sure, see https://github.com/seaweedfs/seaweedfs/pull/6376 . I'll rebase this PR once the change is in.